arch/src/aarch64/fdt.rs

```diff
@@ -587,7 +587,7 @@ mod tests {
         .collect();
 
         let kvm = hypervisor::kvm::KvmHypervisor::new().unwrap();
-        let hv: Arc<dyn hypervisor::Hypervisor> = Arc::new(kvm);
+        let hv: Arc<dyn hypervisor::HypervisorTrait> = Arc::new(kvm);
```

Comment:
As explained in other comments, I don't think we need to re-export `KvmHypervisor` as `Hypervisor`, therefore we shouldn't need to rename the `Hypervisor` trait into `HypervisorTrait` trait.
Also, we should never use `Trait` in a trait's name, as it is redundant.

Reply:
I agree. It is just that Rust (rightfully) wouldn't allow me to export a trait and a concrete type with the same name. This issue may become moot once we settle other issues.